cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/view/AwsGroupView.java

```diff
@@ -54,16 +54,14 @@
 
     private final int onDemandPercentage;
 
-    private final Boolean fastEbsEncryptionEnabled;
-
     private final Double spotMaxPrice;
 
     private final String placementGroupStrategy;
 
     public AwsGroupView(Integer instanceCount, String type, String flavor, String groupName, Boolean ebsEncrypted, Integer rootVolumeSize,
             Map<String, Long> volumeCounts, List<SecurityRule> rules, List<String> cloudSecurityIds, String subnetId, Boolean kmsKeyDefined,
             String kmsKey, String encryptedAMI, boolean useNetworkCidrAsSourceForDefaultRules, String instanceProfile, int onDemandPercentage,
-            boolean fastEbsEncryptionEnabled, Double spotMaxPrice, String placementGroupStrategy) {
```

Comment:
is this change backward compatible ?

Reply:
An instance of AwsGroupView in created in `cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/CloudFormationTemplateBuilder.java` which has been updated along with its test. Is there some other repo I should consider checking this?

Reply:
@doktoric  This view is supposedly only used to populate the FreeMarker context. The FreeMarker templates `aws-cf-stack-freeipa.ftl` and `aws-cf-stack.ftl` have been also updated below and not to depend on the `fastEbsEncryptionEnabled` flag any more. Did you mean if this change is compatible with existing clusters created by an older CB? Even in those cases I think the FreeMarker template is only used when rendering the initial CF template for the LaunchConfiguration; cluster upscaling etc. will probably use the existing CF LaunchConfiguration / LaunchTemplate of the ASG, will not it?

Reply:
true @lajosrodek